<ui version="4.0" >
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>DLD hardware Simulator</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>28</y>
     <width>357</width>
     <height>434</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" >
    <item>
     <widget class="QGroupBox" name="groupBox" >
      <property name="title" >
       <string>Node data</string>
      </property>
      <layout class="QGridLayout" name="gridLayout" >
       <item row="0" column="1" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>X Coord.</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>Y Coord.</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" >
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>Radius</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Node &amp;1: </string>
         </property>
         <property name="buddy" >
          <cstring>node1XSpin</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Node &amp;2: </string>
         </property>
         <property name="buddy" >
          <cstring>node2XSpin</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Node &amp;3: </string>
         </property>
         <property name="buddy" >
          <cstring>node3XSpin</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QDoubleSpinBox" name="node1XSpin" >
         <property name="minimum" >
          <double>-65535.000000000000000</double>
         </property>
         <property name="maximum" >
          <double>65535.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2" >
        <widget class="QDoubleSpinBox" name="node1YSpin" >
         <property name="minimum" >
          <double>-65535.000000000000000</double>
         </property>
         <property name="maximum" >
          <double>65535.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="3" >
        <widget class="QDoubleSpinBox" name="node1RadiusSpin" >
         <property name="minimum" >
          <double>-65535.000000000000000</double>
         </property>
         <property name="maximum" >
          <double>65535.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QDoubleSpinBox" name="node2XSpin" >
         <property name="minimum" >
          <double>-65535.000000000000000</double>
         </property>
         <property name="maximum" >
          <double>65535.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="2" >
        <widget class="QDoubleSpinBox" name="node2YSpin" >
         <property name="minimum" >
          <double>-65535.000000000000000</double>
         </property>
         <property name="maximum" >
          <double>65535.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="3" >
        <widget class="QDoubleSpinBox" name="node2RadiusSpin" >
         <property name="minimum" >
          <double>-65535.000000000000000</double>
         </property>
         <property name="maximum" >
          <double>65535.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QDoubleSpinBox" name="node3XSpin" >
         <property name="minimum" >
          <double>-65535.000000000000000</double>
         </property>
         <property name="maximum" >
          <double>65535.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="2" >
        <widget class="QDoubleSpinBox" name="node3YSpin" >
         <property name="minimum" >
          <double>-65535.000000000000000</double>
         </property>
         <property name="maximum" >
          <double>65535.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="3" >
        <widget class="QDoubleSpinBox" name="node3RadiusSpin" >
         <property name="minimum" >
          <double>-65535.000000000000000</double>
         </property>
         <property name="maximum" >
          <double>65535.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2" >
      <property name="title" >
       <string>Additional data</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_7" >
         <property name="text" >
          <string>&amp;Update interval: </string>
         </property>
         <property name="buddy" >
          <cstring>updateIntervalSpin</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QSpinBox" name="updateIntervalSpin" >
         <property name="toolTip" >
          <string>update intervall in seconds</string>
         </property>
         <property name="maximum" >
          <number>60</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_8" >
         <property name="text" >
          <string>&amp;Maximum Node range: </string>
         </property>
         <property name="buddy" >
          <cstring>maxNodeRangeSpin</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QDoubleSpinBox" name="maxNodeRangeSpin" >
         <property name="toolTip" >
          <string>ATTENTION: a change to this field will take effect only after restart</string>
         </property>
         <property name="statusTip" >
          <string>ATTENTION: a change to this field will take effect only after restart</string>
         </property>
         <property name="maximum" >
          <double>65535.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_9" >
        <property name="text" >
         <string>Zoom: </string>
        </property>
        <property name="buddy" >
         <cstring>zoomSpin</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="zoomSpin" >
        <property name="minimum" >
         <double>1.000000000000000</double>
        </property>
        <property name="value" >
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGraphicsView" name="nodeGraphView" />
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>357</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionQuit" />
   </widget>
   <addaction name="menu_File" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>462</y>
     <width>357</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <action name="actionQuit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="statusTip" >
    <string>Exit application</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>node1XSpin</tabstop>
  <tabstop>node1YSpin</tabstop>
  <tabstop>node1RadiusSpin</tabstop>
  <tabstop>node2XSpin</tabstop>
  <tabstop>node2YSpin</tabstop>
  <tabstop>node2RadiusSpin</tabstop>
  <tabstop>node3XSpin</tabstop>
  <tabstop>node3YSpin</tabstop>
  <tabstop>node3RadiusSpin</tabstop>
  <tabstop>updateIntervalSpin</tabstop>
  <tabstop>maxNodeRangeSpin</tabstop>
  <tabstop>zoomSpin</tabstop>
  <tabstop>nodeGraphView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
